src/gsi-ncdiag/proc_gsi_ncdiag.py.in

```diff
@@ -99,6 +99,12 @@ ap.add_argument("-g", "--geovals_dir",
                 help="Path to directory to output observations")
 ap.add_argument("-d", "--obsdiag_dir",
                 help="Path to directory to output observations")
+ap.add_argument("-obsbias", "--add_obsbias", default=False,
```

Comment:
I think Unix convention is that "--add_obsbias" is fine but the "-" argument should just be one letter either lowercase or capital. Can we shorten the first arguments to be shorter?

Reply:
I'd suggest, 'b' for obsbias, 'q' for qcvars, and t for testvars. In reality, users will probably use the '--' arguments but these shorter ones will be helpful for quicker typing on the command line.